docs/development-guide.md

```diff
@@ -112,9 +112,9 @@ questions: what changed and why? The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```text
-cephfs: update cephfs resize
+cephfs: update cephfs expand
```

Comment:
I think we can leave it, no need to update this is specific to cephfs command not the CSI specification. can you revert it back?

Reply:
@Madhu-1 done. ptal